### PR TITLE
create fnd_classes assignment

### PR DESCRIPTION
**Pull request by CharisseL**
@chortlehoort exercises for fnd-classes. Please take a look and let me know if it needs more added to it.

### CODE REVIEW

File: 01-foundations/exercises/FND_CLASSES.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```
+mkdir -p ~/workspace/python/exercises/classes && cd $_
+touch employees.py
+```
+
+## Instructions
+
+Create a class that contains information about employees of a company and define methods to get employee name, job title, and start date.
+    ```
```
Comment (chortlehoort) [inline 01-foundations/exercises/FND_CLASSES.md]: Put another line break above this or else the Github Markdown parser won't render it correctly.

File: 01-foundations/exercises/FND_CLASSES.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    class Company(object):
+
+        def __init__(self, name, title, start_date):
+            self.name = name
+            self.title = title
+            self.start_date = start_date
+
+        def getName(self):
+            return self.name
+
```
Comment (chortlehoort) [inline 01-foundations/exercises/FND_CLASSES.md]: @CharisseL Put a comment here. Something like

`# Add the remaining methods to full the requirements above`

**Comment by CharisseL** [comment]
@chortlehoort made edits from your line notes for review.

**Comment by chortlehoort** [comment]
👍

**Pull request merged**